### PR TITLE
Google api

### PR DESCRIPTION
**Pull request by RazTamir**

### CODE REVIEW

File: ocs_ci/framework/pytest_customization/marks.py [hunk 1]
```diff
@@ -44,6 +45,12 @@
 # under development, you can mark it with @run_this and run pytest -m run_this
 run_this = pytest.mark.run_this
 
+# Skipif marks
+google_api_required = pytest.mark.skipif(
+    not os.path.exists(os.path.expanduser(
+        config.RUN['google_api_secret'])
+    ), reason="Google API credentials doesn't exists"
```
Comment (ebenahar) [inline ocs_ci/framework/pytest_customization/marks.py]: don't exist

**Comment by RazTamir** [comment]
Verified:

========================================================================================== PASSES ===========================================================================================
------------------------------------------------------------- generated html file: file:///home/ratamir/git/ocs-ci/report.html --------------------------------------------------------------
================================================================================== short test summary info ==================================================================================
PASSED tests/e2e/test_io_performance.py::TestIOPerformance::test_run_io[1GB-rw-1-120-4-cell_to_update0]
PASSED tests/e2e/test_io_performance.py::TestIOPerformance::test_run_io[1GB-rw-6-120-16-cell_to_update1]
PASSED tests/e2e/test_io_performance.py::TestIOPerformance::test_run_io[1GB-rw-12-120-32-cell_to_update2]
================================================================================ 3 passed in 792.21 seconds =================================================================================

And also check this OCS FIO spreadsheet (column B is true data from FIO):
https://docs.google.com/spreadsheets/d/1fgzpOhJoDRnRV_YuEumyDfFi14Aqw_hIEXsWqUOg_8c/edit?usp=sharing

File: ocs_ci/framework/conf/default_config.yaml [hunk 2]
```diff
@@ -21,6 +21,7 @@ RUN:
   cli_params: {}  # this will be filled with CLI parameters data
   client_version: '4.1.4'
   bin_dir: './bin'
+  google_api_secret: '~/.ocs-ci/google_api_secret.json'
```
Comment (prsurve) [inline ocs_ci/framework/conf/default_config.yaml]: Do we have a doc that states what should we have in this file

File: ocs_ci/framework/pytest_customization/marks.py (hunk 1 above)
Comment (RazTamir) [reply ocs_ci/framework/pytest_customization/marks.py]: done

File: tests/e2e/test_io_performance.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                *['1GB', 'rw', 6, 120, 16, (13, 2)],
+                #marks=pytest.mark.polarion_id("OCS-558")
+            ),
+            pytest.param(
+                *['1GB', 'rw', 12, 120, 32, (21, 2)],
+                #marks=pytest.mark.polarion_id("OCS-559")
+            ),
+        ]
+    )
+    def test_run_io(
```
Comment (ebenahar) [inline tests/e2e/test_io_performance.py]: giving this new test, do we still need tests.manage.test_pvc_deletion_during_io.TestDeletePVCWhileRunningIO#test_run_io?

Comment (RazTamir) [reply tests/e2e/test_io_performance.py]: No, I will remove it

File: ocs_ci/framework/conf/default_config.yaml (hunk 2 above)
Comment (RazTamir) [reply ocs_ci/framework/conf/default_config.yaml]: We will.
For now, it will skip in case this file is missing

Comment (zmc) [reply ocs_ci/framework/conf/default_config.yaml]: Shouldn't we just skip the uploading if the file is missing, and still perform the tests?

Comment (RazTamir) [reply ocs_ci/framework/conf/default_config.yaml]: We actually thought about it and there's no point in running performance test if you don't have the data to analyze so skip will save execution time

**Comment by RazTamir** [comment]
don't merge yet please

**Comment by RazTamir** [comment]
Verified:

========================================================================================== PASSES ===========================================================================================
================================================================================== short test summary info ==================================================================================
PASSED tests/e2e/test_io_performance.py::TestIOPerformance::test_run_io[1GB-rw-1-60-4-1]
PASSED tests/e2e/test_io_performance.py::TestIOPerformance::test_run_io[1GB-rw-6-60-16-2]
PASSED tests/e2e/test_io_performance.py::TestIOPerformance::test_run_io[1GB-rw-12-60-32-3]
================================================================================ 3 passed in 613.41 seconds =================================================================================

Results updated here https://docs.google.com/spreadsheets/d/1fgzpOhJoDRnRV_YuEumyDfFi14Aqw_hIEXsWqUOg_8c/edit#gid=0

**Pull request merged**